packages/web/src/components/nav/desktop/NavHeader.tsx

```diff
@@ -98,13 +99,15 @@ export const NavHeader = () => {
         />
       </Link>
       <Flex justifyContent='center' alignItems='center'>
-        <RestrictedLink to={DASHBOARD_PAGE} restriction='account'>
-          <NavHeaderButton
-            icon={IconDashboard}
-            aria-label={messages.dashboardLabel}
-            isActive={location.pathname === DASHBOARD_PAGE}
-          />
-        </RestrictedLink>
+        {accountUser?.track_count ? (
```

Comment:
I think track count doesn't include hidden tracks, so we need to figure out something else ... we probably have this issue elsewhere so not blocking